xml/cap_depl_ldap.xml

```diff
@@ -0,0 +1,162 @@
+<chapter version="5.0" xml:id="cha.cap.ldap"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+  <title>LDAP Integration</title>
+ </info>
+ <para>
+  &productname; can be integrated with <link xlink:href="https://docs.cloudfoundry.org/uaa/identity-providers.html">identity providers</link> to help manage authentication of users. The Lightweight Directory Access Protocol (LDAP) is an example of an identity provider that &cap; integrates with. This section describes the necessary components and steps in order to configure the integration. See <link xlink:href="https://docs.cloudfoundry.org/uaa/uaa-user-management.html">User Account and Authentication LDAP Integration</link> for more information.
+ </para>
+ <sect1 xml:id="sec.cap.ldap-prereqs">
+  <title>Prerequisites</title>
+  <para>
+   The following prerequisites are required in order to complete an LDAP integration with &productname;.
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     The Cloud Foundry command line interface (&cfcli;). (See <xref linkend="sec.cap.cf-cli"/>.)
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <link xlink:href="https://docs.cloudfoundry.org/uaa/uaa-user-management.html">The Cloud Foundry UAA command line interface (UAAC)</link>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     An LDAP server and the credentials for a user/service account with permissions to search the directory.
+    </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+ <sect1 xml:id="sec.cap.ldap-integration">
+  <title>Example LDAP Integration</title>
+  <para>
+   Run the following commands to complete the integration of your &cap; deployment and LDAP server.
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Use UAAC to target your UAA server.
+    </para>
+<screen>&prompt.user;uaac target <replaceable>https://uaa.example.com:2793</replaceable></screen>
+   </listitem>
+   <listitem>
+    <para>
+     Authenticate to the UAA server as <literal>admin</literal> using the <literal>UAA_ADMIN_CLIENT_SECRET</literal> set in your <filename>scf-config-values.yaml</filename> file.
+    </para>
+<screen>&prompt.user;uaac token client get admin -s <replaceable>password</replaceable></screen>
+   </listitem>
+   <listitem>
+    <para>
+     Create the LDAP identity provider. A 201 response will be returned when the identity provider is successfully created. See the <link xlink:href="http://docs.cloudfoundry.org/api/uaa/version/4.21.0/index.html#ldap">UAA API Reference</link> and <link xlink:href="https://github.com/cloudfoundry/uaa/blob/master/docs/UAA-LDAP.md">&cf; UAA-LDAP Documentation</link>for information regarding the request parameters and additional options available to configure your identity provider.
```

Comment:
Do we want to use `https://github.com/cloudfoundry/uaa/blob/4.21.0/docs/UAA-LDAP.md` for the `UAA-LDAP Documentation` instead, to pin it to a version (in case they remove/delete the file in the future)?